nearuplib/nodelib.py

```diff
@@ -146,16 +146,19 @@ def print_staking_key(home_dir):
 def run_binary(path,
                home,
                action,
-               verbose=None,
+               verbose=False,
                shards=None,
                validators=None,
                non_validators=None,
                boot_nodes=None,
                output=None):
     command = [path, '--home', home]
 
-    if verbose or verbose == '':
-        command.extend(['--verbose', verbose])
+    env = os.environ.copy()
+    if verbose:
```

Comment:
Actually we want to have something a bit different here, aka the same logic we have in the neard

--verbose is gonna print all the logs including actix_web and other modules
--verbose module=level should be printing only that specified logging  Example --verbose actix_web=info

Backtrace should be always included by default, so maybe having a --no-backtrace flag for that?